dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/RecordTest.java

```diff
@@ -0,0 +1,14 @@
+package org.gjwatts.liberty;
+
+/**
+ * @author GregoryWatts
```

Comment:
I would add copyright to this file and remove author tag.

Reply:
Done.  Thanks, good catch!